docs/parallelism.md

```diff
@@ -0,0 +1,31 @@
+# Limiting parallelism
+
+You can restrict the number of simultaneous workflow executions.
+
+## Controller level
+
+You can limit the total number of workflows that can execute at any one time in the [workflow controller ConfigMap](./workflow-controller-configmap.yaml).
+
+```yaml
+data:
+  parallelism: "10"
+```
+
+You can also limit the number of workflows that can execute in a single namespace.
+
+```yaml
+data:
+  namespaceParallelism: "4"
+```
+
+Workflows that are executing but restricted from running more nodes due to the other mechanisms will still count towards the parallelism limits.
+
+### Priority
```

Comment:
Should this be called "Queueing" to match the "Synchronization" page?

```suggestion
### Queueing
```

Reply:
I also wonder if we should just link to the "Synchronization" page, as, if I'm not mistaken, it's effectively identical here

Reply:
The queue is an implementation detail from the users perspective. All they care about is that priority exists and they can use it to prioritise. They don't need to know this is implemented by queueing.

Reply:
I made that comment before your change to link them, so let me re-read the new variant